src/modules/ekf2/EKF2.cpp

```diff
@@ -2405,6 +2405,15 @@ void EKF2::UpdateGpsSample(ekf2_timestamps_s &ekf2_timestamps)
 			return; //TODO: change and set to NAN
 		}
 
+		if (fabsf(_param_ekf2_gps_yaw_off.get()) > 0.f) {
+			if (!PX4_ISFINITE(vehicle_gps_position.heading_offset) && PX4_ISFINITE(vehicle_gps_position.heading)) {
+				// Convert the offset to radians & appy offset
+				float raw_yaw_offset = matrix::wrap_pi(_param_ekf2_gps_yaw_off.get());
```

Comment:
Convert to radians using wrap_pi? :D

Reply:
I was originally converting to radians explicitly: https://github.com/PX4/PX4-Autopilot/pull/23821#discussion_r1822945299

What change are you requesting? Add back in the radians conversion?

Reply:
removed the radians comment

Reply:
`EKF2_GPS_YAW_OFF` [is in degrees](https://github.com/PX4/PX4-Autopilot/pull/23821/files#diff-de800df88a6874ed094f72d78d51c1c061851ce99d07fd2497788f5fb82a2ee8R54-R56), you need to convert it to radians first and then apply the `wrap_pi`:
```suggestion
				float raw_yaw_offset = matrix::wrap_pi(math::radians(_param_ekf2_gps_yaw_off.get()));
```

The comment from @dagar was to change `_params->gps_yaw_off` to `_param_ekf2_gps_yaw_off.get()`, not to remove the conversion to radians.